### ISSUE TITLE
Jupyter server Error 500: Internal Server Error when opening Matlab notebook

### ISSUE THREAD

**Issue by Hiroo-Miyata** [mathworks/jupyter-matlab-proxy#19]
# Source Error
```
[E 22:54:16.507 NotebookApp] Uncaught exception GET /matlab/ (::1)                                                      
HTTPServerRequest(protocol='http', host='localhost:8888', method='GET', 
uri='/matlab/', version='HTTP/1.1', remote_ip='::1')                                                                                                              
Traceback (most recent call last):                                                                                     
File "C:\Users\chaselab\miniconda3\lib\site-packages\tornado\web.py", line 1713, 
in _execute                           result = await result                                                                                              
File "C:\Users\chaselab\miniconda3\lib\site-
packages\jupyter_server_proxy\websocket.py", line 91, in get               return await 
self.http_get(*args, **kwargs)                                                                        File 
"C:\Users\chaselab\miniconda3\lib\site-
packages\jupyter_server_proxy\handlers.py", line 683, in http_get          return 
await ensure_async(self.proxy(self.port, path))                                                             
File "C:\Users\chaselab\miniconda3\lib\site-packages\jupyter_server\utils.py", line 
182, in ensure_async               result = await obj                                                                                                 
File "C:\Users\chaselab\miniconda3\lib\site-
packages\jupyter_server_proxy\handlers.py", line 677, in proxy             await 
self.ensure_process()                                                                                        File 
"C:\Users\chaselab\miniconda3\lib\site-
packages\jupyter_server_proxy\handlers.py", line 654, in ensure_process                                                                                                                         
await proc.start()                                                                                                 File 
"C:\Users\chaselab\miniconda3\lib\site-packages\simpervisor\process.py", line 90, 
in start                        self.proc = await asyncio.create_subprocess_exec(                                                                  
File "C:\Users\chaselab\miniconda3\lib\asyncio\subprocess.py", line 236, in 
create_subprocess_exec                     transport, protocol = await 
loop.subprocess_exec(                                                                  File 
"C:\Users\chaselab\miniconda3\lib\asyncio\base_events.py", line 1676, in 
subprocess_exec                          transport = await 
self._make_subprocess_transport(                                                                 File 
"C:\Users\chaselab\miniconda3\lib\asyncio\base_events.py", line 498, in 
_make_subprocess_transport                raise NotImplementedError                                                                                        
NotImplementedError   
```

# Environment
windows 10
miniconda latest version: Conda 4.12.0 Python 3.9.7
followed README.md
the same error is happened by other person (https://stackoverflow.com/questions/74042907/jupyter-server-error-500-internal-server-error-when-opening-matlab-notebook)

**Comment by prabhakk-mw** [comment]
Dear @Hiroo-Miyata 

Thank you for the detailed information.
The windows support for `jupyter-matlab-proxy` is gated on an issue with `jupyter-server-proxy` which is a package we depend on.
More information on the issue can be viewed in this [Pull Request](https://github.com/jupyterhub/jupyter-server-proxy/pull/181).

From the looks of it, the PR has a suggested workaround to additionally install [jupyter-server-proxy-windows](https://pypi.org/project/jupyter-server-proxy-windows/) to work around the issue.

I haven't personally tested this workaround and cannot speak to its efficacy.
Please update this issue with any information if you attempt the suggested workaround.

**Comment by rashedmyt** [comment]
Duplicate of #35

**Comment by prabhakk-mw** [comment]
Dear @Hiroo-Miyata 

As of v0.6.0 of `jupyter-matlab-proxy`, the package should now support Windows. 
Please let us know if you face any issues with using it.
